client/src/pages/Configurations/ConfigBuild/index.tsx

```diff
@@ -55,17 +59,21 @@ export default function ConfigBuild() {
         </StepsContainer>
       </NavigationContainer>
       <SectionContainer>
-        <Builder code_sets={response.data.code_sets} />
+        <Builder
+          id={response.data.id}
+          code_sets={response?.data.code_sets ?? []}
```

Comment:
We should always receive an array back in `code_sets`.
```suggestion
          code_sets={response?.data.code_sets}
```